src/index.html

```diff
@@ -1,17 +1,163 @@
+          >
+        </label>
+      
+        <label for="dateOfBirth" class="form-field">
+          Full date of birth:
+          <input
+            type="date"
+            name="dateOfBirth"
+            value="mm/dd/yyyy"
+          >
+        </label>
+      
+        <label for="terms">I accept the term of the agreement
+          <input type="checkbox" name="terms">
+        </label>
+      </fieldset>
+
+      <fieldset class="groupField">
+        <legend>Registration:</legend>
+        <label for="email" class="form-field">
+          E-mail:
+          <input 
+          type="email" 
+          name="email" 
+          placeholder="email@example.com" 
+          required
+          >
+        </label>
+      
+        <label for="password">
+          Password:
+          <input 
+          type="password" 
+          name="password" 
+          required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="groupField">
+        <legend>An interesting fact about you!</legend>
+        <div class="form-field">
+          Do you love cats?
+          <label>
+            <input type="radio" name="cat">
+            Yes
+          </label>
+          <label>
+            <input type="radio" name="cat">
+            No
+          </label>
+        </div>
+        <label for="color" class="form-field">
+          What is your favorite color?
+          <input
+          type="color" 
+          name="color" 
+          value="#000000"
+          >
+        </label>
+
+        <label for="timeToBed" class="form-field">
+          What time do you go to bed?
+          <input 
+          type="time" 
+          name="timeToBed" 
+          step="2"
+          >
+        </label>
+      
+        <label for="carBrand" class="form-field">
+          What are your favorite brands of cars?
+          <select name="carBrand" multiple>
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="lada">lada</option>
+          </select>
+        </label>
+      
+        <label for="rate">
+          How do you rate our work?
+          <input 
+          type="range" 
+          name="rate" 
+          min="0" 
+          max="100"
+          value="0"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="groupField">
+        <legend>Additional info:</legend>
+        <label for="comments" class="form-field">
+        Comments:
+          <textarea 
+            class="comment" 
+            name="comments" 
+            minlength="50" 
+            maxlength="300"
+         ></textarea>
+        </label>
+      
+        <label for="reccomend">
+          Would you reccomend us?
+          <select name="reccomend">
+            <option value="yes">yes</option>
+            <option value="no">no</option>
+          </select>
+        </label>
+      </fieldset>
+
+      <button type="submit">Submit</button>
+
+    </form>
```

Comment:
Don't add spaces between parent and child components.
```suggestion
      <button type="submit">Submit</button>
    </form>
```